index.bs

```diff
@@ -869,6 +871,19 @@ its inheritance hierarchy has a cycle.  That is, an interface
 |A| cannot inherit from itself, nor can it inherit from another
 interface |B| that inherits from |A|, and so on.
 
+<div algorithm>
```

Comment:
I assume this can't just say "_I_ and _I_'s **inherited interfaces**" because we want a list, not a set?

Reply:
Yeah. I've got a note on my todo list to see if we can get back to just one concept.